app/src/main/java/com/amaze/filemanager/asynchronous/asynctasks/MoveFiles.java

```diff
@@ -188,6 +206,13 @@ public void onPostExecute(Boolean movedCorrectly) {
             });
 
         } else {
+
+            if (destinationSize < totalBytes) {
+                // destination don't have enough space; return
+                publishProgress(context.getResources().getString(R.string.in_safe));
```

Comment:
This is a hack, no *progress* is being published. I would have said "please use onPostExcecute()", but this is onPostExcecute(), please simply move the Toast creation here (and remember to remove collateral changes).